biogrid/libs/biogrid-simulator/src/lib/building/building.ts

```diff
@@ -78,8 +77,8 @@ export class Building implements EnergyUser {
     this.buildingId = Id;
   }
 
-  getEnergyInJoules(): number {
-    return this.energyInJoules;
+  getEnergyInKilowattHour(): number {
+    return this.energyKilowatt;
```

Comment:
variable name and getter name should match, so I recommend changing the variable name to `this.energyInKilowattHour`